.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "game/reveal.js"]
```

Comment:
Using gitmodules is not straightforward. It would be better to add the repo directly in the folder

Reply:
Also it will fix the wrong paths to files if different reveal.js version